### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by kewartoscar** [zauberzeug/nicegui#2367] [Q&A] [answered]
### Question

Is there a website with the list of all icons that can be used with NiceGui?

**Comment by falkoschindler** [comment, marked as answer]
There's a link in the [documentation](https://nicegui.io/documentation/icon):

> [Here](https://fonts.google.com/icons?icon.set=Material+Icons) is a reference of possible names.

**Reply by kewartoscar** [reply]
Thanks a million.
